src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesV1Spec.scala

```diff
@@ -226,6 +226,29 @@ class ResourceRoutesV1Spec extends FlatSpec with Matchers with ScalatestRouteTes
     }
   }
 
+  "GET /api/resources/v1/{resourceType}/{resourceId}/actions" should "200 on list resource actions" in {
+    val resourceType = ResourceType(ResourceTypeName("rt"), Set(ResourceActionPattern("run", "", false)), Set(ResourceRole(ResourceRoleName("owner"), Set(ResourceAction("run")))), ResourceRoleName("owner"))
+    val samRoutes = TestSamRoutes(Map(resourceType.name -> resourceType))
+
+    Post(s"/api/resources/v1/${resourceType.name}/foo") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.NoContent
+    }
+
+    Get(s"/api/resources/v1/${resourceType.name}/foo/actions") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.OK
+      responseAs[Set[String]]
```

Comment:
do you mean to do some assert on the response?

Reply:
no, this only checks that it can be parsed

Reply:
Shall we check the actions with expected ones? or is it not feasible to do that?

Reply:
the purpose of the test is to check that the route exists, returns the right status code and the right format of data. Correctness of that data is tested by a service level test.